hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import {
+    Card,
+    CardMedia,
+    Select,
+    FormControl,
+    MenuItem,
+    IconButton,
+    Paper,
+    InputLabel,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+export const CartCard = ({
```

Comment:
After reading the Material UI Card API, I think it's best to structure your code like this:

```javascript
<Card>
    <CardMedia />
    <CardContent>
          {put all the typography and form dropdown inside here}
    </CardContent>
    <CardActions>
           <IconButton>
                    <CloseIcon />
           </IconButton>
      </CardActions>
</Card>
```

So remove Paper and Grid as well